setup.py

```diff
@@ -42,10 +42,11 @@ def find_version(*file_paths):
 
     author = 'Iskandarov Eduard',
     author_email = 'e.iskandarov@corp.mail.ru',
```

Comment:
Given @toidi said this email is no longer active, I would suggest it be removed.